docs/fsharp/get-started/install-fsharp.md

```diff
@@ -49,7 +49,7 @@ You must also install the [.NET Core SDK](https://www.microsoft.com/net/download
 
 ### [Windows](#tab/windows)
 
-If you're on Windows, you must [install Visual Studio with F# support](#install-f-with-visual-studio). This installs all the necessary components to write, compile, and execute F# code.
+[Install Visual Studio with F# support](#install-f-with-visual-studio) if you're on Windows. This installs all the necessary components to write, compile, and execute F# code.
```

Comment:
conditional sentence should come first

also probably would look better if the verb Install was out of the link